discord/ext/pages/pagination.py

```diff
@@ -75,7 +76,7 @@ def __init__(
             style=style,
             disabled=disabled,
             custom_id=custom_id,
-            row=0,
+            row=row or 0,
```

Comment:
why not just default row to 0?

Reply:
because I was working too quickly and based it on the code in `update()` (which does require a default of `None`). I've fixed it now.